elizhahn/niche-audience
# Feature/remove shelf item

## What does this PR do (summary of changes)?
- A user can now click the X next to each item and remove that item from their shelf
- A delete method was created which uses the item's id and shelf category to make an api call. 
- The shelf needed to be replaced using an updated list, because the api did not have an api endpoint to delete an item inside of their "baskets" 
## How should it be tested?
- Add a few items in a few categories, click the X, the item should be removed dynamically and the user should see it be removed from their list
- refresh the page, all items should still be removed
## Future Iterations:
- style the remove button
